### PR TITLE
chore: BP-27783 | parse boolean args, use yarn frozen lock

### PR DESCRIPTION
**Pull request by littleninja**
- Parse boolean args to support `false` (`'false'` string) or other falsy arguments
- Update yarn install to use frozen lockfile

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -85,7 +88,7 @@ async function run() {
       // Yarn
       if (!skipInstall) {
         core.startGroup('Install dependencies');
-        await exec.exec('yarn');
+        await exec.exec('yarn', ['install', '--frozen-lockfile']);
```
Comment (mvarrieur) [inline src/main.ts:91]: (nit): You really don't need to change this but just a heads up you can run `yarn --frozen-lockfile` and skip the `install` part

**Review by mvarrieur — APPROVED** [review]
🎉

File: src/main.ts (hunk 1 above)
Comment (littleninja) [reply src/main.ts]: Noted, removed

**Review by mvarrieur — APPROVED** [review]
:shipit:

**Comment by littleninja** [comment]
I don't have merge permission and am mobile. Do you @mvarrieur or @szechyjs ?

**Pull request merged**